spring-xsuaa/src/test/java/com/sap/cloud/security/xsuaa/token/TokenAuthenticationConverterTest.java

```diff
@@ -33,6 +34,8 @@ public void setup() {
 
 		tokenConverterLocalScopesOnly = new TokenAuthenticationConverter(xsAppName);
 		tokenConverterLocalScopesOnly.setLocalScopeAsAuthorities(true);
+
+		tokenConverterOauth2 = new OAuth2AuthenticationConverter(new DefaultAuthoritiesExtractor());
```

Comment:
pls create a new Test Class OAuth2AuthenticationConverterTest and move the code here into this one.

Reply:
ok